notebook/static/notebook/js/actions.js

```diff
@@ -423,6 +415,21 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
+        'toggle-toolbar':{
+            help: 'hide/show the toolbar',
+            handler : function(env){
+                $('div#maintoolbar').toggle();
```

Comment:
Might as well fix it while we are looking at it - @ellisonbg can you change it to `env.toolbar.element.toggle()` please?